server/src/test/scala/za/co/absa/atum/server/aws/AwsSecretsProviderSpec.scala

```diff
@@ -48,7 +48,7 @@ class AwsSecretsProviderSpec extends ConfigProviderSpec {
           awsSecretValue <- AwsSecretsProvider.getSecretValue(awsConfig.dbPasswordSecretName)
         } yield assertTrue(dummySecretValue == awsSecretValue)
       }
-    )
+    ) @@ TestAspect.ifPropNotSet("runIntegration")
```

Comment:
Not an IT.

Reply:
Addressed in commit - a5b5b61dee1304ceadbcb2174fceb585ea95d38f.